### PR TITLE
Python 3.11

### PR DESCRIPTION
**Pull request by ThibaudDauce**
Fix https://github.com/datagouv/data.gouv.fr/issues/1357

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
Have you tested it a bit to see that there isn't any side effect?

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: udata/circleci:2-alpine
+      - image: udata/circleci:py3.11-bookworm
```
[the remark this answers is not in the document]
Comment (ThibaudDauce) [reply .circleci/config.yml:7]: I used the latest from https://hub.docker.com/r/udata/circleci/tags (I think the OS is newer? but need to see with @jordanguedj )

**Comment by ThibaudDauce** [comment]
> Have you tested it a bit to see that there isn't any side effect?

I tried locally it seems to work…

**Review by maudetes — APPROVED** [review]
Don't forget the changelog entry :)

**Pull request merged**